app/models/post.rb

```diff
@@ -11,6 +11,13 @@ class Post < ApplicationRecord
   validates :body, presence: true, length: { maximum: 140 }
   validate :allowed_picture_content_type
 
+  def self.yesterday_liked_ranking_top_ten
+    now = Time.current
+    yesterday_range = now.yesterday.beginning_of_day...now.yesterday.end_of_day
+    select('posts.*',
+           'count(likes.id) AS likes').left_joins(:likes).where(likes: { created_at: yesterday_range }).group('posts.id').order('likes DESC').limit(10)
```

Comment:
likes_count にしておいたほうが混乱しないかもとおもった。
```suggestion
           'count(likes.id) AS likes_count').left_joins(:likes).where(likes: { created_at: yesterday_range }).group('posts.id').order('likes_count DESC').limit(10)
```